docs/examples/eventrouter.yaml

```diff
@@ -113,24 +147,28 @@ spec:
                           image: gcr.io/heptio-images/eventrouter:v0.2
                           imagePullPolicy: IfNotPresent
                           volumeMounts:
-                          - name: config-volume
-                            mountPath: /etc/eventrouter
+                            - name: config-volume
+                              mountPath: /etc/eventrouter
                       serviceAccount: event-router-sa
                       volumes:
                         - name: config-volume
                           configMap:
                             name: event-router-cm
-
     delete:
       template: |
         apiVersion: argoproj.io/v1alpha1
         kind: Workflow
+        metadata:
+          name: er-delete-
```

Comment:
This name will be ignored I believe we create our own name for the workflows [here](https://github.com/keikoproj/addon-manager/blob/master/controllers/addon_controller.go#L392-L399)

Reply:
Agree, you are right, we are generating names. 
But, any k8s resource lacks either of grv/kind/name would be a risky, and pokes potential issues.